src/output_gstreamer.c

```diff
@@ -577,12 +577,22 @@ static int output_gstreamer_init(void)
 	return 0;
 }
 
+static const char *output_gstreamer_version(char *buffer, size_t len)
+{
+	snprintf(buffer, len, "%s (glib-%d.%d.%d; gstreamer-%d.%d.%d)",
```

Comment:
I think I'd be also fine if the module would return a statically allocated string. We could either have a static allocated buffer and snprintf() into it, or just assemble the stirng with macro tricks.

Providing less headache for the reader of the calling code is a good thing.

Also, it keeps our API interface lean. It is not that we have a heavy lifting function here that requires some external megabyte buffer to be filled, but something that for essentially all known implementation will return a static string in the first place.